processor/tests/test_bin_process.py

```diff
@@ -226,3 +226,16 @@ def _validate(fs, server_env):
 
     _validate(gcsfs_a(), server_a_env)
     _validate(gcsfs_b(), server_b_env)
+
+    # assert that bad-id is not in the final published set
+    paths = gcsfs_a().walk(server_a_env["BUCKET_INTERNAL_PRIVATE"])
+    processed = [p for p in paths if "processed" in p]
+    raw = [p for p in paths if "raw" in p]
+
+    # query contains bad-id and other valid ids
```

Comment:
should this be "raw query"?

Reply:
I was using query in the sense that the following assertions are queries on the path list e.g. "assert any path in raw contains `batch_id=bad-id`". I reworded the comments so they align with the variables a bit more.